### PR TITLE
Dit 555

### PR DESCRIPTION
**Pull request by retroredge**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -115,7 +115,7 @@ mongodb {
 scheduler {
   data-migration {
     lock-lifetime = 60s
-    interval = 15s
+    interval = 1s
```
Comment (ed0906) [inline conf/application.conf]: Intentional?

File: test/unit/uk/gov/hmrc/bindingtariffadminfrontend/model/MigratableCaseTest.scala [hunk 2]
```diff
@@ -41,7 +41,8 @@ class MigratableCaseTest extends UnitSpec with MockitoSugar {
         Some("queue"),
         application,
         Some(decision),
-        Seq.empty
+        Seq.empty,
+        Set.empty
```
Comment (ed0906) [inline test/unit/uk/gov/hmrc/bindingtariffadminfrontend/model/MigratableCaseTest.scala]: I appreciate the one above is a `Seq.empty` but we should probably test the mapping of at least one keyword

**Comment by ed0906** [comment]
Looks fine

File: test/unit/uk/gov/hmrc/bindingtariffadminfrontend/model/MigratableCaseTest.scala (hunk 2 above)
Comment (retroredge) [reply test/unit/uk/gov/hmrc/bindingtariffadminfrontend/model/MigratableCaseTest.scala]: Adding some test data for keyword sets throughout the tests

**Pull request merged**